.github/workflows/linux.yml

```
@@ -68,7 +68,7 @@ jobs:
         mkdir build
         cd build
         cmake -D CMAKE_BUILD_TYPE=Debug \
-              -D DEAL_II_CXX_FLAGS='-Werror' \
+              -D DEAL_II_CXX_FLAGS='-Werror -std=c++17' \
```

We could also use the CMAKE functionality. I'm actually undecided what I prefer personally :-)
```suggestion
              -D CMAKE_CXX_STANDARD=17 \
              -D DEAL_II_CXX_FLAGS='-Werror' \
```

---

I have changed this!